client/packages/invoices/src/Returns/DetailView/Footer/Footer.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { FC, memo } from 'react';
+import {
+  Box,
+  ButtonWithIcon,
+  StatusCrumbs,
+  XCircleIcon,
+  useTranslation,
+  AppFooterPortal,
+  //   InvoiceNodeStatus,
+  useBreadcrumbs,
+  InvoiceNodeStatus,
+} from '@openmsupply-client/common';
+import { getStatusTranslator, outboundStatuses } from '../../../utils';
+import { OutboundReturnRowFragment, useReturns } from '../../api';
+
+// import { useOutbound, OutboundFragment } from '../../api';
+import { StatusChangeButton } from './StatusChangeButton';
+import { OnHoldButton } from './OnHoldButton';
+
+const createStatusLog = (invoice: OutboundReturnRowFragment) => {
+  const statusIdx = outboundStatuses.findIndex(s => invoice.status === s);
+  const statusLog: Record<InvoiceNodeStatus, null | undefined | string> = {
+    [InvoiceNodeStatus.New]: null,
+    [InvoiceNodeStatus.Allocated]: null,
+    [InvoiceNodeStatus.Picked]: null,
+    [InvoiceNodeStatus.Shipped]: null,
+    [InvoiceNodeStatus.Delivered]: null,
+    [InvoiceNodeStatus.Verified]: null,
+  };
+  if (statusIdx >= 0) {
+    statusLog[InvoiceNodeStatus.New] = invoice.createdDatetime;
+  }
+  if (statusIdx >= 1) {
+    statusLog[InvoiceNodeStatus.Allocated] = invoice.allocatedDatetime;
+  }
```

Comment:
Can remove this step, `Allocated` isn't a status for Outbound Returns - probably also means you should create an `outboundReturnsStatuses` for use on line 21, and in the component below 👍 

Have added in the naming issue to rename all the existing `outbound` references to `outboundShipment` so stuff like this is less easy to confuse!

Reply:
Yup, I've done it the same way as it's currently done in InboundShipment -- the unused one is still in this object, but it's removed from the proper list of statuses: 1c8f1168a

This will make it easier for abstracting this function to a common one too.